region/maridia/inner-pink.json

```diff
@@ -1004,6 +1165,143 @@
+                        "HiJump",
+                        {"or":[
+                          "canCrouchJump",
+                          "canDownGrab"
+                        ]}
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "G-Mode Overload PLMs - PB the Items (To the Left Door)",
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,5,6],
+                      "mode": "direct",
+                      "artificialMorph": false
+                    }},
+                    {"or":[
+                      {"itemNotCollectedAtNode": 7},
+                      {"itemNotCollectedAtNode": 8}
+                    ]},
+                    "Morph",
+                    {"ammo": { "type": "PowerBomb", "count": 2 }},
+                    "h_canNavigateUnderwater",
+                    "canSnailClimb",
+                    {"or":[
+                      "Gravity",
+                      "h_canMaxHeightSpringBallJump",
+                      {"and":[
+                        "HiJump",
+                        {"or":[
+                          "canCrouchJump",
+                          "canDownGrab"
+                        ]}
+                      ]}
+                    ]}
+                  ],
+                  "reusableRoomwideNotable": "Aqueduct G-Mode Overload PLMs - PB the Items",
+                  "note": [
+                    "PLMs can be overloaded in direct G-Mode with a single Power Bomb if both items are still there and 2 PBs if only one item is.",
+                    "There is a row of tiles that works, just above and to the left of the right door.",
+                    "The row is one tile higher than the horizontal pipe that is part of the door frame, spanning from the left to one tile away from the right wall."
+                  ],
+                  "devNote": "This does not include canRiskPermanentLossOfAccess, as it is only worth doing this strat if the items are there."
+                },
+                {
+                  "name": "G-Mode Morph Overload PLMs - PB the Items (To the Left Door)",
+                  "notable": true,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2,5,6],
+                      "mode": "direct",
+                      "artificialMorph": true
+                    }},
+                    {"or":[
+                      {"itemNotCollectedAtNode": 7},
+                      {"itemNotCollectedAtNode": 8}
+                    ]},
+                    "SpringBall",
+                    {"ammo": { "type": "PowerBomb", "count": 2 }},
+                    "h_canNavigateUnderwater",
+                    "canSnailClimb",
+                    {"or":[
+                      "Gravity",
+                      "h_canMaxHeightSpringBallJump",
```

Comment:
```suggestion
```

Reply:
With morph you take a different strat and that makes this option just confusing.